### DISCUSSION TITLE
crypto-simple-earn/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#3] [Ideas]
# crypto-simple-earn/

首先得非常感谢 @taresky 的文章 「无风险年化 360%？小白也能懂的 Crypto 套利」，让许多人可以以一个非偏见的形式「了解逻辑，理解风险，明白利润的来源」。 在文

https://nova.moe/crypto-simple-earn/

**Comment by boris1993** [comment]
Hi，麻烦问下现在OKX那个 simple earn 的API还能用吗？我刚刚直接拷到浏览器里打开，只有JSON结构， `data.all` 是个空数组。

**Reply by n0vad3v** [reply]
可以的，可能和你访问 IP 有关，我记得当时我写的时候某些 IP 访问就会返回空数组。
![Screenshot from 2024-04-01 23-40-11](https://github.com/n0vad3v/blog-comments/assets/24852034/56d46ea9-a14c-4f53-ac00-e03abedca74e)

**Reply by boris1993** [reply]
感谢答疑，看来我需要强制走梯子才行

**Comment by wenweih** [comment]
博主你好，我转载了您的文章，已在文末表明出处，如果觉得不适合的话还请麻烦您联系删文。谢谢

https://metanethub.com/column_articles/9102d549f9

**Reply by n0vad3v** [reply]
好的，我的文章准循 CC BY-NC 4.0 协议，可以在给出适当的署名的情况下转载。
看了一下转载的文章带上了 “原文链接” ，建议可以加个署名~

**Reply by wenweih** [reply]
感谢，标题追加上 Nova Kwok，并在文末加上博主主页。
